### ISSUE TITLE
"change color" wrongly translated to german

### ISSUE THREAD

**Issue by aoloe** [scratchblocks/scratchblocks#277]
```
change [color v] effect by(25)
```

gets translated to

```
ändere [Farbe v] -Effekt um (25) :: looks 
```

but the scratch UI uses:

```
ändere Effekt [Farbe v]  um (25)
```

**Comment by aoloe** [comment]
on top of it,

```
ändere Effekt [Farbe v]  um (25) :: looks
```

is correctly rendered, while 

```
ändere Effekt [Farbe v]  um (25)
```

is not.

no idea if this is ok so...

**Comment by tjvr** [comment]
Thanks for reporting! Perhaps we just need to update the translations...

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by tjvr** [comment]
Oh interesting, looks like `scratchr2_translations` which we're currently using is now deprecated, in favour of `scratch-l10n`. That could be a pain to update :disappointed:

**Comment by tjvr** [comment]
This should be fixed by #285 -- please open a new issue if there are still problems. :)
